src/test/java/Homework21.java

```diff
@@ -0,0 +1,28 @@
+import org.example.pages.HomePage;
+import org.example.pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest {
+    String newPlayListName = "New playlist";
+    String currentPlayListName = "TestPro Playlist";
+    HomePage homePage = null;
+
+    @Test
+    public void renamePlayListTest() throws InterruptedException {
+        LoginPage loginPage = new LoginPage(getDriver());
+        loginPage.login("demo@class.com", "te$t$tudent");
+        homePage = new HomePage(getDriver());
+        homePage.renamePlayList(currentPlayListName, newPlayListName);
+
+        Assert.assertEquals(newPlayListName, homePage.getPlayListByName(newPlayListName).getText());
+        //Assert.assertTrue(homePage.getPlaylistByName(newPlayListName).isDisplayed());
+    }
+
+    @AfterMethod
+    public void rollBackChanges() throws InterruptedException {
+        homePage = new HomePage(getDriver());
+        homePage.renamePlayList(newPlayListName, currentPlayListName);
+    }
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

Reply:
deleted